### PR TITLE
[Android] Control library directory from upper level

### PR DESCRIPTION
**Pull request by lincsoon**
Specify the directory from where the infrastructure loads native
libraries from the package org.xwalk.core instead of
org.xwalk.core.internal. This is to support for the lite mode and future
download mode in which native libraries don't exist in the default
directory.

### CODE REVIEW

**Comment by lincsoon** [comment]
@axinging @fujunwei @XiaosongWei

File: runtime/android/core/src/org/xwalk/core/XWalkLibraryInterface.java [hunk 1]
```diff
@@ -24,22 +24,29 @@
...
     /**
-     * The Crosswalk signature verification failed.
+     * The version of the Crosswalk runtime is newer than the application.
      */
-    public static final int STATUS_SIGNATURE_CHECK_ERROR = 4;
+    public static final int STATUS_NEWER_VERSION = 4;
 
     /**
-     * The version of the Crosswalk runtime is older than the application.
+     * Mismatch of CPU Architecture for the Crosswalk Runtime.
```
Comment (XiaosongWei) [inline runtime/android/core/src/org/xwalk/core/XWalkLibraryInterface.java]: forgot to update the comment?

**Comment by XiaosongWei** [comment]
LGTM with a comment concern.

**Comment by fujunwei** [comment]
LGTM

**Pull request merged**